### DISCUSSION TITLE
在modal组件中使用Image组件，预览时候顶部的操作栏丢失问题

### DISCUSSION THREAD

**Discussion by dengxiwang** [ant-design/ant-design#39044] [Q&A] [answered]
更改预览下的这个参数内容
<img width="304" alt="image" src="https://user-images.githubusercontent.com/41525895/205048368-d30c05b9-4ec3-4a0c-89ca-0b87620e7401.png">

**Comment by MadCcc** [comment, marked as answer]
Fixed in https://github.com/react-component/image/pull/151
